src/vivarium_testing_utils/automated_validation/comparison.py

```diff
@@ -323,10 +323,10 @@ def _align_datasets(
         ## At this point, the only non-common index levels should be scenarios and draws.
         return stratified_test_data, aggregated_reference_data
 
-    def _aggregate_strata_reference(
-        self, data: pd.DataFrame, strata: Collection[str] = ()
+    def _aggregate_reference_stratifications(
+        self, data: pd.DataFrame, stratifications: Collection[str] = ()
     ) -> pd.DataFrame:
-        for stratum in strata:
+        for stratum in stratifications:
```

Comment:
```suggestion
        for stratification in stratifications:
```
Also need to update the error message below